cmd/version.go

```diff
@@ -24,16 +25,70 @@ func versionString() string {
 	return v
 }
 
-func getCmdVersion(_ *state.GlobalState) *cobra.Command {
-	// versionCmd represents the version command.
-	return &cobra.Command{
+type versionCmd struct {
+	gs     *state.GlobalState
+	isJSON bool
+}
+
+func (c *versionCmd) run(cmd *cobra.Command, _ []string) error {
+	if !c.isJSON {
+		root := cmd.Root()
+		root.SetArgs([]string{"--version"})
+		_ = root.Execute()
+		return nil
+	}
+
+	details := consts.VersionDetails()
+	if exts := ext.GetAll(); len(exts) > 0 {
+		type extInfo struct {
+			Module  string   `json:"module"`
+			Version string   `json:"version"`
+			Imports []string `json:"imports"`
+		}
+
+		ext := make(map[string]extInfo)
+		for _, e := range exts {
+			if v, ok := ext[e.Path]; ok {
+				v.Imports = append(v.Imports, e.Name)
```

Comment:
nitpick: I guess this is impossible, but still might be good idea to check th e version also matches

Reply:
We discussed it a bit with @szkiba, currently they all have the same version, but I could implement this check in advance

Reply:
Done in 1854d82aa7c15db3f212d59912630a209b042181